version

```diff
@@ -1,2 +1,2 @@
 # Use Semantic versioning only. Please update the version number before opening a pull request.
-v1.0.1
+v1.0.2
```

Comment:
This isn't a patch, it's a minor (new feature)

Reply:
Done